Product/Production/Common/Properties/src/main/resources/exchangeInfo.xml

```diff
@@ -310,6 +319,15 @@
                 </ns2:endpointConfiguration>
               </ns2:endpointConfigurationList>
             </ns2:endpoint>
+            <ns2:endpoint>
+              <ns2:name>DocumentDataSubmission</ns2:name>
```

Comment:
Need only one endpoint DocumentDataSubmission or DocDataSubmission. Please confirm on the service name as well.